doc/user/content/guides/cdc-postgres.md

```diff
@@ -0,0 +1,260 @@
+
+_Create views for specific tables included in the Postgres publication:_
+
+```sql
+CREATE VIEWS FROM SOURCE mz_source (view1, view2);
+```
+
+_Create views for all tables:_
+
+```sql
+CREATE VIEWS FROM SOURCE mz_source;
+```
+
+Under the hood, Materialize parses this statement into view definitions for each table (so you don't have to!).
+
+### Create a materialized view
+
+Any materialized view defined on top of this source will be incrementally updated as new change events stream in, as a result of `INSERT`, `UPDATE` and `DELETE` operations in the original Postgres database.
+
+```sql
+CREATE MATERIALIZED VIEW cnt_view1 AS
+SELECT field1,
+       COUNT(*) AS cnt
+FROM view1
+GROUP BY field1;
+```
+
+## Kafka + Debezium
+
+If Kafka is part of your stack, you can use [Debezium](https://debezium.io/) and the [Kafka source](/sql/create-source/avro-kafka/#debezium-envelope-details) to propagate CDC data from Postgres to Materialize. Debezium captures row-level changes resulting from `INSERT`, `UPDATE` and `DELETE` operations in the upstream database and publishes them as events to Kafka using Kafka Connect-compatible connectors.
+
+### Setup
+
+**Minimum requirements:** Postgres 10+
+
+Before deploying a Debezium connector, you need to ensure that the upstream database is configured to support [logical replication](https://www.postgresql.org/docs/10/logical-replication.html). As a _superuser_:
+
+1. Check the [`wal_level` configuration](https://www.postgresql.org/docs/current/wal-configuration.html) setting:
+
+    ```sql
+    SHOW wal_level;
+    ```
+
+    The default value is `replica`. For CDC, you'll need to set it to `logical` in the database configuration file (`postgresql.conf`). Keep in mind that changing the `wal_level` requires a restart of the Postgres instance and can affect database performance.
+
+    **Note:** Additional steps may be required if you're using Postgres on [Amazon RDS](https://debezium.io/documentation/reference/1.6/connectors/postgresql.html#postgresql-on-amazon-rds).
+
+1. Grant enough privileges to ensure Debezium can operate in the database. This will depend on how much control you want to give to the replication user, so we recommend following the [Debezium documentation](https://debezium.io/documentation/reference/connectors/postgresql.html#postgresql-replication-user-privileges).
+
+1. Set the replica identity value for any tables that you want to replicate but have **no primary key** defined:
+
+    ```sql
+    ALTER TABLE repl_table REPLICA IDENTITY FULL;
+    ```
+
+    This setting determines the amount of information that is written to the WAL in case of `UPDATE` and `DELETE` operations. By setting it to `FULL`, change events will include the previous values of all the table’s columns; as a heads-up, you should expect a performance hit in the database from increased CPU usage.
```

Comment:
"in the case of" can almost always be a single preposition -- in this case "in".

Same rewrite on the sentences following as above.